src/app/screens/Enable/index.tsx

```diff
@@ -30,6 +31,20 @@ function Enable(props: Props) {
     event.preventDefault();
   }
 
+  async function block(event: React.MouseEvent<HTMLAnchorElement>) {
+    msg.error(
+      "User added site to blocklist domain, host " +
```

Comment:
Can [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) be used?